orc8r/cloud/go/mproto/protos/mproto.proto

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package magma.orc8r.mproto;
+option go_package = "protos";
+
+// --------------------------------------------------------------------------
+// A set of protobuf messages, all individually encoded & paired with unique identifiers
+// --------------------------------------------------------------------------
```

Comment:
- Don't use these dividor-type comment lines, just put the docstring comments directly
- Use golang-style doc comments https://magma.github.io/magma/docs/next/contributing/contribute_conventions#follow-style-conventions
```suggestion
// ProtoSlice is a set of protobuf messages, all individually encoded and paired with unique identifiers
```